asab/log.py

```diff
@@ -446,13 +446,23 @@ def _on_write(self):
 		self._write_ready = True
 		self._loop.remove_writer(self._socket)
 
+		error_counter = 0
```

Comment:
Add a comment to `on_write` -> this method is a contingency dump when the socket is not ready.